tools/hicup/.shed.yml

```diff
@@ -2,9 +2,14 @@ name: hicup
 owner: bgruening
 description: The HiCUP-Pipeline from the Bioinformatics Babraham Institute.
 
-long_description: A galaxy wrapper implementation of the HiCUP-Pipeline from the Bioinformatics Babraham Institute. See http://www.bioinformatics.babraham.ac.uk/projects/hicup/.
-  
+long_description: A galaxy wrapper implementation of the HiCUP-Pipeline from the Bioinformatics Babraham Institute. See http://www.bioinformatics.babraham.ac.uk/projects/hicup/ 
+
 
 categories:
   - Epigenetics
-remote_repository_url: https://github.com/joachimwolff/galaxytools/tree/hicup/tools/hicup
\ No newline at end of file
+remote_repository_url: https://github.com/joachimwolff/galaxytools/tree/hicup/tools/hicup
+
```

Comment:
Here is something missing to instruct planemo to do the demultiplexing isn't it? Please compare with other examples.